frontend/store/wallet.ts

```diff
@@ -61,19 +60,13 @@ export const actions = {
             dispatch('authorizations/reset', undefined, { root: true });
         }
     },
-    async connect({ commit, rootGetters, dispatch }: ActionContext<State, State>, walletType: string): Promise<void> {
+    async connect({ commit, dispatch }: ActionContext<State, State>, walletType: string): Promise<void> {
         dispatch('authorizations/reset', undefined, { root: true });
         commit('setIsLoading', true);
         try {
             const wallet = getWallet(walletType);
             await wallet.connect();
             commit('setAddress', wallet.address);
-            if (wallet.address) {
```

Comment:
What changed now, that wallet will always have an address?

Reply:
No, the call within `if` is no longer needed, since the way we "attach" wallet to app changed